package.json

```diff
@@ -11,9 +11,9 @@
   "scripts": {
     "start": "start-storybook -p 6006",
     "watch": "babel src -w -d dist --ignore src/stories",
-    "build": "babel src -d dist --ignore src/stories",
+    "build": "rimraf dist && tsc",
     "build:storybook": "build-storybook -o .out",
-    "test": "npm run lint && mocha --compilers js:babel-core/register test/*.spec.js",
+    "test": "npm run lint && mocha --require ts-node/register test/*.spec.js",
```

Comment:
If we're no longer using `babel` at all, can we drop it from the dev dependencies?

Reply:
We need to keep our presets, as they are used by storybook, but I think everything else can be removed.